tests/rptest/tests/partition_balancer_test.py

```diff
@@ -75,6 +75,51 @@ def node2partition_count(self):
 
         return ret
 
+    def get_partition_amount_on_node(self, node):
+        admin = Admin(self.redpanda)
+        return len(
+            list(
+                filter(lambda x: x["ns"] == "kafka",
+                       admin.get_partitions(node=node))))
+
+    def validate_metrics_node_down(self, disabled_node,
+                                   expected_moving_partitions_amount):
+        availabale_nodes = list(
+            filter(lambda x: x != disabled_node, self.redpanda.nodes))
+        metrics = self.redpanda.metrics_sample(
+            "moving_to_node",
+            availabale_nodes,
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        moving_to_node_partitions_amount = int(
+            sum([metric.value for metric in metrics.samples]))
+        if moving_to_node_partitions_amount != expected_moving_partitions_amount:
+            return False
+
+        metrics = self.redpanda.metrics_sample(
+            "moving_from_node",
+            availabale_nodes,
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        moving_from_node_partitions_amount = int(
+            sum([metric.value for metric in metrics.samples]))
+        if moving_from_node_partitions_amount != 0:
+            return False
+
+        metrics = self.redpanda.metrics_sample(
+            "node_cancelled_movements",
+            availabale_nodes,
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        cancelled_movements_for_node = int(
+            sum([metric.value for metric in metrics.samples]))
+        if cancelled_movements_for_node != 0:
+            return False
+
+        return True
+
+    def check_metrics(self, disabled_node, expected_moving_partitions_amount):
+        return wait_until(lambda: self.validate_metrics_node_down(
```

Comment:
Is it easy to debug this? I think we at least should debug-log somewhere the metrics values. Or maybe they are logged somewhere already by `metrics_sample`?